azure-pipelines.yml

```diff
@@ -20,13 +20,17 @@ stages:
   jobs:
   - job: 'Test'
     pool:
-      vmImage: 'Ubuntu-16.04'
+      vmImage: 'Ubuntu-latest'
     strategy:
         matrix:
           Python36:
```

Comment:
Let's drop support for Python36 already :+1: Can remove this line

Reply:
Added in following push :ok_hand: